aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,134 @@
+
+:::image type="content" source="maui/_static/new-project.png" alt-text="Choose a template.":::
+
+> [!NOTE]
+> If the **.NET MAUI App (Preview)** project template does not appear in the **Create a new project** window, open the Developer Command Prompt for VS 2022 Preview and run the following command: `devenv.exe /updateConfiguration`. Then, restart Visual Studio.
+
+In the **Configure your new project** window, name your project, choose a suitable location for it, and click the **Create** button:
+
+:::image type="content" source="maui/_static/configure-project.png" alt-text="Configure the project.":::
+
+Wait for Visual Studio to create the project and for the project's dependencies to be restored:
+
+:::image type="content" source="maui/_static/restored-dependencies.png" alt-text="Restored dependencies.":::
+
+In the **Android SDK License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/android-sdk-license.png" alt-text="Android SDK License Acceptance window.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+Wait for Visual Studio to download the Android SDK and Android Emulator.
+
+## Run the app in the Windows Machine Emulator
+
+In the Visual Studio toolbar, select the **Windows Machine** button to build the app:
+
+:::image type="content" source="maui/_static/windows-machine-button.png" alt-text="Windows Machine button.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
+If Developer Mode isn't enabled, you're prompted to enable it in **Settings** > **For developers** > **Developer Mode**. Set the switch to **On**:
+
+:::image type="content" source="maui/_static/windows-developer-mode.png" alt-text="Windows Developer Mode enabled.":::
+
+The running app in the Windows Machine Emulator:
+
+:::image type="content" source="maui/_static/running-app-windows-1.png" alt-text="App running in the Windows Machine Emulator.":::
+
+Navigate to the **`Counter`** page. Select the **`Click me`** button to increment the counter:
+
+:::image type="content" source="maui/_static/running-app-windows-2.png" alt-text="Counter page running in the Windows Machine Emulator.":::
+
+## Run the app in the Android Emulator
+
+In the Visual Studio toolbar, select the **Android Emulator** button to build the app:
+
+:::image type="content" source="maui/_static/android-emulator-button.png" alt-text="Android Emulator button.":::
+
+In the **User Account Control** dialog, select the **Yes** button:
+
+:::image type="content" source="maui/_static/user-account-control.png" alt-text="User Account Control dialog.":::
+
```

Comment:
Much of the "roughness" 🐶 that I mentioned is from directly copying over what they had in the .NET MAUI tutorial. This was what they posted, and I vaguely remember hitting it along the way, but I'd have to double-check it. btw - same deal with it for the Windows Machine option. I removed that step/image from the tutorial per your remark, but I think I did hit it yesterday.

It's their **Step 8** of ...

https://docs.microsoft.com/en-us/dotnet/maui/get-started/first-app#get-started-with-visual-studio-2022-171-preview

I'll leave it for the moment, but you'll either need to confirm removal or in order for me to get it again, it looks like I would need to uninstall VS 2022 and reinstall it (*maybe* 🤔).

Reply:
I may not be seeing the UAC dialogs because I've been using Windows Sandbox to test in a clean environment, and I think Windows Sandbox hides the UAC dialogs. Even so, I don't really think it's necessary to call out the UAC dialogs in the tutorial.

Reply:
Ok ... I'll drop them.